app/kubernetes/converters/namespace.js

```diff
@@ -13,19 +15,45 @@ class KubernetesNamespaceConverter {
     res.Yaml = yaml ? yaml.data : '';
     res.ResourcePoolName = data.metadata.labels ? data.metadata.labels[KubernetesPortainerResourcePoolNameLabel] : '';
     res.ResourcePoolOwner = data.metadata.labels ? data.metadata.labels[KubernetesPortainerResourcePoolOwnerLabel] : '';
+    res.NamespaceSystemLabel = data.metadata.labels ? data.metadata.labels[KubernetesPortainerNamespaceSystem] : undefined;
```

Comment:
this property name looks like it contains the "label" instead of its value. maybe `NamespaceSystemLabelValue`?

do we really need this value saved?